bioblend/galaxy/histories/__init__.py

```diff
@@ -167,16 +234,12 @@ def update_dataset(self, history_id, dataset_id, **kwds):
 
         :type history_id: str
         :param history_id: Encoded history ID
-        :type name: str
-        :param name: Replace history dataset name with the given string
-        :type annotation: str
-        :param annotation: Replace history dataset annotation with given string
-        :type deleted: bool
-        :param deleted: Mark or unmark history dataset as deleted
-        :type visible: bool
-        :param visible: Mark or unmark history dataset as visible
```

Comment:
Hmmm. I think I removed them because they weren't explicit function parameters. That was wrong of me, I should've added them as extra arguments.